README.md

```diff
@@ -1,10 +1,10 @@
-# Draft Content Guide
+# GOV.AU Content Guide
 
 This guide is designed to help people working in the Australian Government create simpler, clearer and faster information.
 
-It can be currently accessed at http://content-style-guide.apps.staging.digital.gov.au/
+Live site: https://guides.service.gov.au/content-guide/
 
-**Status:** the guide is in active development.
+Staging site: http://content-style-guide.apps.staging.digital.gov.au/
```

Comment:
Hmmm I've been using http://content-guide.apps.staging.digital.gov.au/ - I think http://content-style-guide.apps.staging.digital.gov.au/ is your old staging site?